.github/workflows/cd.yml

```diff
@@ -65,7 +65,7 @@ jobs:
     - uses: actions/checkout@v2
     - uses: actions/setup-python@v2
       with:
-        python-version: '3.9'
+        python-version: '3.8'
```

Comment:
Why *reduce* the Python version?

Reply:
Because it matches my local python version and I thought it might be related (no issues building locally for me). You did make a good point with the conditional above though. I will investigate this and see if we can't bump back up to python 3.9

Reply:
or 3.10 ;)